wiki/Disambiguation/Rank/de.md

```diff
@@ -0,0 +1,14 @@
+---
+tags:
+  - grade
+  - ranking
+  - Benotung
+---
+
+# Rank (Begriffserklärung)
+
+**Rank** kann Folgendes bedeuten:
+
+- Eine [Benotung](/wiki/Grade)
+- Der [Spielerrang](/wiki/Ranking) oder eine Position auf den Online-Ranglisten
+- Das Verb "zu ranken", das den Durchlauf durch das [Beatmap Ranking-Verfahren](/wiki/Beatmap_ranking_procedure) meint
```

Comment:
```suggestion
- Das Verb "zu ranken", das den Durchlauf durch das [Beatmap-Ranking-Verfahren](/wiki/Beatmap_ranking_procedure) meint
```
Not sure if necessary, but I personally would put a hyphen there. Feel free to dismiss this one, though.

Reply:
The German article about the [osu!mania ranking criteria](https://osu.ppy.sh/wiki/de/Ranking_Criteria/osu!mania) uses no hyphens at all. I find using hypens in this case better, so lets use this.